### PR TITLE
[PD-1901] job count cache

### PR DESCRIPTION
**Pull request by mklauber**
Ready for merge, imo.  See my second comment asking if anyone else can think of a reason why that would need to be delayed.

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -788,6 +788,7 @@ def task_update_solr(jsid, **kwargs):
 def task_etl_to_solr(guid, buid, name):
     try:
         import_jobs.update_job_source(guid, buid, name)
+        BusinessUnit.clear_cache(int(buid))
```
Comment (mklauber) [inline tasks.py]: This is the new functionality

File: tasks.py [hunk 2]
```diff
@@ -798,7 +799,7 @@ def task_etl_to_solr(guid, buid, name):
 def task_priority_etl_to_solr(guid, buid, name):
     try:
         import_jobs.update_job_source(guid, buid, name)
-        task_clear_bu_cache.delay(buid=int(buid), countdown=1500)
+        BusinessUnit.clear_cache(int(buid))
```
Comment (mklauber) [inline tasks.py]: Any reason to delay this further, rather than running immediately after indexing?

File: tasks.py (hunk 1 above)
Comment (aspidites) [inline tasks.py]: so buid for import_jobs.update_job_source needs to be a string and for BusinessUnit.clear_cache needs to be an integer?

File: tasks.py (hunk 2 above)
Comment (aspidites) [inline tasks.py]: I tried to git blame to see when that delay was added, but this file has moved around enough that it was hard for me to tell.

**Pull request merged**